platform/windows/export/export.cpp

```diff
@@ -253,37 +253,76 @@ Error EditorExportPlatformWindows::_rcedit_add_data(const Ref<EditorExportPreset
 	String rcedit_path = EditorSettings::get_singleton()->get("export/windows/rcedit");
 
 	if (rcedit_path != String() && !FileAccess::exists(rcedit_path)) {
-		add_message(EXPORT_MESSAGE_WARNING, TTR("Resources Modification"), vformat(TTR("Could not find rcedit executable at \"%s\"."), rcedit_path));
+		add_message(EXPORT_MESSAGE_WARNING, TTR("Resources Modification"),
+				vformat(TTR("Could not find rcedit executable at \"%s\"."), rcedit_path));
 		return ERR_FILE_NOT_FOUND;
 	}
 
+	// If no explicit path, try from PATH
 	if (rcedit_path == String()) {
-		rcedit_path = "rcedit"; // try to run rcedit from PATH
+		rcedit_path = "rcedit";
+
+		// --- Preflight Test ---
+		int pre_exit = 0;
+		List<String> help_args;
+		help_args.push_back("--help");
+
+		// child_id=nullptr, pipe=nullptr, exitcode=&pre_exit, read_stderr=false
+		Error pre_res = OS::get_singleton()->execute(
+				rcedit_path, help_args, true, nullptr, nullptr, &pre_exit, false);
+
+		if (pre_res != OK || pre_exit != 0) {
+			add_message(EXPORT_MESSAGE_WARNING, TTR("Resources Modification"),
+					TTR("No rcedit found in PATH. Configure rcedit in the Editor Settings (Export > Windows > rcedit), or install it so it is available in PATH."));
+			return ERR_FILE_NOT_FOUND;
+		} else {
+			print_line("rcedit preflight check passed via PATH.");
+		}
```

Comment:
This addition make no sense and will always fail on non Windows platforms, since it's not using Wine.